src/resource-extensions/resource-extensions.js

```diff
@@ -69,16 +69,17 @@
           'delete': { method: 'DELETE' }
         };
 
-        var factoryArgs = [];
-        for ( var i = 0; i < 3; i++ ) {
-          factoryArgs.push( arguments[i] || {} );
-        }
-        factoryArgs[2] = angular.extend( factoryArgs[2], DEFAULT_ACTIONS, ( ngynResourceProvider.actions || {} ) );
-        var actions = factoryArgs[2];
-        var actionKeys = Object.keys( actions );
-        var resourceResult = $delegate.apply( this, factoryArgs );
-
-        angular.forEach( actionKeys, function( action ) {
+        // following angular's convention of extending the passed in actions with the defaults,
+        // we collect the full action set at this point, because they all need to be wrapped in the loop below
+        var actions = angular.extend( {}, DEFAULT_ACTIONS, arguments[2], ( ngynResourceProvider.actions || {} ) );

```

Comment:
This is the significant change - we no longer extend the original args, and we change the order the defaults are applied.

This fixes an issue where the resource defines a new verb, but is overwritten by the angular default